File: patito/python/1245.py
# Problem ID: 1245
# Title: De arriba hacia abajo

x, y = map(int, input().split())
may = 0
men = 0
if x > y:
    may = x
    men = y
else:
    may = y
    men = x
while may >= men:
    print(may)
    may -= 1

File: patito/java/1245.java
// Problem ID: 1245
// Title: De arriba hacia abajo

import java.util.Scanner;
public class Main
{
	public static void main(String[] args) {
	    Scanner sc = new Scanner(System.in);
	    int x, y, may, men;
	    x = sc.nextInt();
	    y = sc.nextInt();
	    if( x > y ){
	        may = x;
	        men = y;
	    }else{
	        may = y;
	        men = x;
	    }
	    while(may >= men){
	        System.out.println(may);
	        may -= 1;
	    }
	}
}